uber/common.py

```diff
@@ -29,23 +29,18 @@
 from io import StringIO, BytesIO
 from itertools import chain, count
 from collections import defaultdict, OrderedDict
-from urllib.parse import quote, urlparse, parse_qsl
+from urllib.parse import quote, urlparse, parse_qsl, quote_plus
```

Comment:
I don't expect other people to maintain my bizarre "the line lengths of imports in ``common.py`` must be in ascending order of line length" convention.  Just know that I will go in and "fix" this later if you don't :P